var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -62,7 +62,12 @@ class Perl(Package):
 
     def install(self, spec, prefix):
         configure = Executable('./Configure')
-        configure("-des", "-Dprefix=" + prefix)
+        configure_args = ["-des", "-Dprefix=" + prefix]
+        # Discussion of -fPIC for Intel at:
+        # https://github.com/LLNL/spack/pull/3081
+        if spec.satisfies('%intel'):
+            configure_args.append("-Accflags=" + self.compiler.pic_flag)
+        configure(*configure_args)
```

Comment:
Thanks!  But... I was hoping it would go at the beginning of the class, like this:
```
# Author: George Hartzell <hartzell@alerce.com>
# Date: July 21, 2016
# Author: Justin Too <justin@doubleotoo.com>
# Date: September 6, 2015
#
from spack import *


class Perl(Package):   # Perl does not use Autotools, it should subclass from Package
    """Perl 5 is a highly capable, feature-rich programming language with over
       27 years of development."""
```